review-demo.rb

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env ruby
+
+# Add the lib directory to Ruby's load path so our requires will work.
+$:.unshift(File.expand_path('./lib'))
+
+require 'happo'
+
+system 'open', Happo::Utils.construct_url('/review-demo')
+require 'happo/server'
```

Comment:
I did, but I wanted it to be available when developing in the repo without having to install the gem and this seemed like the easiest way to make that possible.